### ISSUE TITLE
Running face swap and i'm not getting a swapped face..

### ISSUE THREAD

**Issue by vrgamegirl19** [Gourieff/comfyui-reactor-node#398]
### First, confirm

- [X] I have read the [instruction](https://github.com/Gourieff/comfyui-reactor-node/blob/main/README.md) carefully
- [X] I have searched the existing issues
- [X] I have updated the extension to the latest version

### What happened?

when i try and run the fast face swap it goes through the process and finishes but the image is still the main input image. It does not swap the face.  see attached snippet with settings.
![image](https://github.com/user-attachments/assets/9af6d38a-8247-4308-92f9-318da45dfac9)

### Steps to reproduce the problem

[workflowMyFaceSwap.json](https://github.com/user-attachments/files/16701092/workflowMyFaceSwap.json)

### Sysinfo

windows 11, rtx 3080ti

### Relevant console log

```Shell
no logs
```

**Comment by gamaraala666** [comment]
Hi first of all, in my personal experience, the gender detection is notoriously fickle, or maybe I'm also missing something. 
But I just downloaded your flow changed both to no, and it works as it should. 
To my eyes that last line of the error log indicates somethings wrong with insightface. So, till someone more knowledgeable comes along, I would advice you to look into your insightface installation. 
If you already had a working build and this just cropped up all of a sudden, probably a recent update got you. in that case roll back or wait for the next update.

**Comment by vrgamegirl19** [comment]
> or the next update.

I have never been able to get it to work. Today I even got a new hard drive, did a fresh install of comfyUI and it still won't work!! I have tried reinstalling insight face over and over again and still can't get it working. gah.

**Comment by gamaraala666** [comment]
Have you tried using the prebuilt wheels?
https://github.com/Gourieff/comfyui-reactor-node?tab=readme-ov-file#troubleshooting

**Comment by embasy** [comment]
> Have you tried using the prebuilt wheels? https://github.com/Gourieff/comfyui-reactor-node?tab=readme-ov-file#troubleshooting

When it says to place the prebuilt wheel into the ComfyUI root folder, which address exactly is it referring to? Just want to make sure I'm putting in the right directory. (Newbie here)

**Comment by gamaraala666** [comment]
If you are using the portable version,
![root folder](https://github.com/user-attachments/assets/651940bc-d4c6-44da-a1cf-28c351b2c0bb)

This is the root folder. 
**Heres a simplified version of the guide for your use case. (Again, I'm assuming you have the portable version)**
1. Right click in the root folder and open a terminal, or navigate to the root folder using the terminal. Check your python version from here using...
`python_embeded\python.exe -V`

2.  Download the appropriate wheel. Then copy it into the root folder. 

3. Upgrade PIP using...
`python_embeded\python.exe -m pip install -U pip`

4. Install insightface using the correct command for your python version...
`python_embeded\python.exe -m pip install insightface-0.7.3-cp310-cp310-win_amd64.whl` (for python 3.10) or `python_embeded\python.exe -m pip install insightface-0.7.3-cp311-cp311-win_amd64.whl` (for python 3.11) or `python_embeded\python.exe -m pip install insightface-0.7.3-cp312-cp312-win_amd64.whl` (for python 3.12).

**Comment by vrgamegirl19** [comment]
> Have you tried using the prebuilt wheels? https://github.com/Gourieff/comfyui-reactor-node?tab=readme-ov-file#troubleshooting

Yes. I have tried this so many times. I even did a fresh install and tried it all over again. Still won't work.

**Comment by vrgamegirl19** [comment]
> If you are using the portable version, ![root folder](https://private-user-images.githubusercontent.com/124975825/363421232-651940bc-d4c6-44da-a1cf-28c351b2c0bb.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxMjM1MDAsIm5iZiI6MTcyNTEyMzIwMCwicGF0aCI6Ii8xMjQ5NzU4MjUvMzYzNDIxMjMyLTY1MTk0MGJjLWQ0YzYtNDRkYS1hMWNmLTI4YzM1MWIyYzBiYi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgzMVQxNjUzMjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYzk2NjUxYzRmZmJhOGU2ZTU1OWUyN2EzNDIxYTI0OGQ4YWQxNWI5NGQ0MmQ2NTNlMmM3NWJmZGJhOGYzZTRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.5vAIeJf2EcnvZe6qJXNUNbRfPnUbRIqxQqm3jNinjL0)
> 
> This is the root folder. **Heres a simplified version of the guide for your use case. (Again, I'm assuming you have the portable version)**
> 
> 1. Right click in the root folder and open a terminal, or navigate to the root folder using the terminal. Check your python version from here using...
>    `python_embeded\python.exe -V`
> 2. Download the appropriate wheel. Then copy it into the root folder.
> 3. Upgrade PIP using...
>    `python_embeded\python.exe -m pip install -U pip`
> 4. Install insightface using the correct command for your python version...
>    `python_embeded\python.exe -m pip install insightface-0.7.3-cp310-cp310-win_amd64.whl` (for python 3.10) or `python_embeded\python.exe -m pip install insightface-0.7.3-cp311-cp311-win_amd64.whl` (for python 3.11) or `python_embeded\python.exe -m pip install insightface-0.7.3-cp312-cp312-win_amd64.whl` (for python 3.12).

I have tried these exact steps many times. Even on a fresh install of comfy on a brand new hard drive. I keep getting the "reshape" error no matter what.  I have friends that got it working and walked me through how they did it and It still won't work for me. I'm guessing there is something in the background that might be causing insight face to not work correctly but I have no idea what it is.

**Comment by Gourieff** [comment]
Please show `python_embeded\python.exe -m pip list`

**Comment by SammyT100** [comment]
Is there any progress right now? I got the same issue. I am gonna be crazy for this thing QQ.

**Comment by lapakkaki8** [comment]
found the Issue

**Comment by lapakkaki8** [comment]
found the issue, I got the same problem and download the missing data from my installation folder from here https://huggingface.co/datasets/Gourieff/ReActor/tree/main/models

**Comment by Aty1111** [comment]
I also encountered the same error message as you, and could not select “no” when using the parameter.
Now I've solved it, so I can give you some reference.
My problem was that the files in the buffalo_l folder were not downloaded in full, so I went to the hf address provided by the author to download them separately, and then unzipped them into the correct file path, and the problem was solved.
